.github/workflows/deploy_tre.yml

```diff
@@ -236,30 +236,30 @@ jobs:
           wkspc_id=`cat workspace_id.txt`
           echo "WORKSPACE_ID=$wkspc_id" >> $GITHUB_ENV
 
-      - name: Clean created test resources
-        shell: bash
-        if: always()
-        env:
-          TF_VAR_tre_id: ${{ secrets.TRE_ID }}
-          WORKSPACE_ID: ${{ env.WORKSPACE_ID }}
-          TF_VAR_location: ${{ secrets.LOCATION }}
-          TF_VAR_address_space: "192.168.25.0/24"
-          TF_VAR_terraform_state_container_name: ${{ secrets.TF_STATE_CONTAINER }}
-          TF_VAR_mgmt_resource_group_name: ${{ secrets.MGMT_RESOURCE_GROUP }}
-          TF_VAR_mgmt_storage_account_name: ${{ secrets.STATE_STORAGE_ACCOUNT_NAME }}
-          AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
-
-        run: |
-          export USE_ENV_VARS_NOT_FILES=true
-          export env
-          export  TF_VAR_workspace_id=$(echo "${WORKSPACE_ID: -4}")
-          export ARM_CLIENT_ID=$(echo "$AZURE_CREDENTIALS" | jq -r '.clientId')
-          export ARM_CLIENT_SECRET=$(echo "$AZURE_CREDENTIALS" | jq -r '.clientSecret')
-          export ARM_SUBSCRIPTION_ID=$(echo "$AZURE_CREDENTIALS" | jq -r '.subscriptionId')
-          export ARM_TENANT_ID=$(echo "$AZURE_CREDENTIALS" | jq -r '.tenantId')
-
-          cd ./workspaces/vanilla/terraform/
-          ../../../devops/scripts/terraform_wrapper.sh -g $TF_VAR_mgmt_resource_group_name -s $TF_VAR_mgmt_storage_account_name  -n $TF_VAR_terraform_state_container_name -k "${TF_VAR_tre_id}-ws-${TF_VAR_workspace_id}" -c "terraform destroy -auto-approve"
+      #- name: Clean created test resources
```

Comment:
Did you mean to delete these - or are they needed for later?

Reply:
deleted